### PR TITLE
Add Intel NuC target

### PR DESCRIPTION
**Pull request by turbank**
Add the intel nuc target as intel-corei7-64
Install all pakages to get FM USB tuner working with genivi demo app

### CODE REVIEW

File: .bbtravis.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ env:
   matrix:
   - TARGET=qemux86-64
   - TARGET=raspberrypi2
+  - TARGET=intel_nuc-core-i5
```
Comment (JeremR) [inline .bbtravis.yml]: intel-corei7-64 ? as this is the machine name and it should work on any intel core based target ? or I am missing something ?

Comment (JeremR) [reply .bbtravis.yml]: Also currently to make CI working (mainly artifactory push) this value must be the same as MACHINE

File: gdp-src-build/conf/templates/intel_nuc-core-i5.local.conf [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+MULTI_PROVIDER_WHITELIST += "virtual/libgl virtual/egl virtual/libgles1 virtual/libgles2"
+PREFERRED_PROVIDER_virtual/libgles1 = ""
+PREFERRED_PROVIDER_virtual/libgles2 = ""
+PREFERRED_PROVIDER_virtual/egl = ""
+PREFERRED_PROVIDER_virtual/libgl = "opengl"
+PREFERRED_PROVIDER_virtual/mesa = ""
+PREFERRED_PROVIDER_libgbm = "libgbm"
+PREFERRED_PROVIDER_libgbm-dev = "libgbm"
+SDKIMAGE_FEATURES_append = " staticdev-pkgs"
+IMAGE_INSTALL_append = " mesa-megadriver qtbase rtl-sdr radioservice alsajackout fmradioqtplugin gstreamer1.0-plugins-good"
```
Comment (JeremR) [inline gdp-src-build/conf/templates/intel_nuc-core-i5.local.conf]: you should create a packagegroup in your radioservice layer

File: gdp-src-build/conf/templates/intel_nuc-core-i5.local.conf [hunk 3]
```diff
@@ -11,5 +11,5 @@ PREFERRED_PROVIDER_virtual/mesa = ""
 PREFERRED_PROVIDER_libgbm = "libgbm"
 PREFERRED_PROVIDER_libgbm-dev = "libgbm"
 SDKIMAGE_FEATURES_append = " staticdev-pkgs"
-IMAGE_INSTALL_append = " mesa-megadriver"
+IMAGE_INSTALL_append = " mesa-megadriver qtbase rtl-sdr radioservice alsajackout fmradioqtplugin gstreamer1.0-plugins-good"
```
Comment (JeremR) [inline gdp-src-build/conf/templates/intel_nuc-core-i5.local.conf]: or maybe you should fix dependencies in your fmradio.bbappend and others in your layer ?

**Review by tardyp — CHANGES_REQUESTED** [review]
yeah! its building. Now I believe a git rebase with clean history is required

**Comment by tardyp** [comment]
why did you close?
You can just do your rebase, and force push to you intel_nuc branch

**Comment by tardyp** [comment]
then you can reopen the PR to have the CI restart

**Pull request closed without merging**